### PR TITLE
Added the Krumware logo & roadmap to README.md

### PR DESCRIPTION
**Pull request by eddie-knight**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,4 +16,42 @@ Build the binaries and copy them to the bin directory
...
+[Common Cloud Controls](https://github.com/finos/common-cloud-controls) project (CCC).
+
+In collaboration with the CFI Reproducible Infrastructure Working Group, this Raid is being run as
+part of a CI pipeline to validate a [CFI-managed RDS service](https://github.com/finos/cfi-ansible-aws-rds)
+deployed by an ansible playbook.
+
+### Near Future
+
+As the CFI Reproducible Infrastructure WG continues to iterate on their infrastructure as code,
+additional tests ("strikes) can be built to ensure holistic compliance with the CCC taxonomy.
```
Comment (grudra7714) [inline README.md:46]: missed ending quote

**Review by grudra7714 — CHANGES_REQUESTED** [review]
minor change

File: README.md (hunk 1 above)
Comment (grudra7714) [reply README.md:46]: `"strikes`

Comment (eddie-knight) [inline README.md:46]:
```suggestion
additional tests ("strikes") can be built to ensure holistic compliance with the CCC taxonomy.
```

Comment (eddie-knight) [reply README.md]: thanks!

**Review by grudra7714 — APPROVED** [review]

**Pull request merged**